src/types/common.ts

```diff
@@ -40,6 +40,12 @@ export type Integer = string;
  */
 export type Usdc = string;
 
+/**
+ * Utility type to help distinguish [[Integer]]s that represent a Wei (18 dec)
+ * value.
```

Comment:
In web v3 repo, we use `Wei` as the equivalent to `Integer` in this repo, meaning it does not represent any number of decimals. Not sure what is the best naming, but we should then stay with one. Since `Integer` is all around the app now, probably best to stay with it?

Reply:
I agree, let's stick to one convention. Refactored with https://github.com/yearn/yearn-sdk/pull/267/commits/89d69677082e61653919f80c9e2516f8e3ad9c63 (introduces the `Unit` type)